syncmaster/worker/__init__.py

```diff
@@ -1,2 +1,20 @@
 # SPDX-FileCopyrightText: 2023-2024 MTS PJSC
 # SPDX-License-Identifier: Apache-2.0
+from celery import Celery
+from sqlalchemy import create_engine
+
+from syncmaster.worker.settings import WorkerAppSettings
+
+
+def create_celery_app(settings) -> Celery:
+    app = Celery(
+        __name__,
+        broker=settings.broker.url,
+        backend="db+" + settings.database.sync_url,
+    )
+    app.settings = settings
+    app.engine = create_engine(url=settings.database.sync_url)
+    return app
+
+
+celery = create_celery_app(WorkerAppSettings())
```

Comment:
Please move under `if __name__ == "__main__":` or to a separated `__main__.py`

Reply:
We actually could create own celery object in server factory function, and then pass it to `/runs` endpoint using dependency injection. Celery requires only database url and broker url which are awailable in ServerAppSettings.

Reply:
may be place TODO here and keep celery in `__init__.py,` as except dependency injection in backend, there should also be:
- logic with initializing celery app in scheduler
- implemented fixtures to overwrite dependency injection with mocks in unit tests

Reply:
Ok, let's add TODO for now